onnxruntime/core/graph/model.cc

```diff
@@ -104,7 +105,27 @@ Model::Model(std::unique_ptr<ModelProto> model_proto, const IOnnxRuntimeOpSchema
 
   std::unordered_map<std::string, int> domain_to_version;
   for (auto& opSet : model_proto_->opset_import()) {
-    domain_to_version[opSet.domain()] = gsl::narrow_cast<int>(opSet.version());
+    const auto& domain = opSet.domain();
+    const auto version = opSet.version();
+    // empty domain and 'ai.onnx' are equivalent
+    if ((domain.empty() || std::string("ai.onnx").compare(domain) == 0) &&
+        version < 7) {
+      if (version == 6) {
```

Comment:
version <= 6. 
also print a warning that says 'please convert your models to opset 7 and support for versions below is not guaranteed.'